tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -62,85 +142,154 @@ def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
 
-        rsh_cmd = f"exec {pod_name} -- tar xvf {pod_path}/tmp/file.gz -C {pod_path}/tmp"
-        _ocp.exec_oc_cmd(rsh_cmd)
+            _ocp = OCP(namespace=pvc_obj.namespace)
 
-        for x in range(copies):
-            rsh_cmd = f"exec {pod_name} -- mkdir -p {pod_path}/folder{x}"
+            rsh_cmd = f"rsync {dir_path} {pod_name}:{pod_path}"
             _ocp.exec_oc_cmd(rsh_cmd)
-            rsh_cmd = f"exec {pod_name} -- cp -r {pod_path}/tmp {pod_path}/folder{x}"
+
+            rsh_cmd = (
+                f"exec {pod_name} -- tar xvf {pod_path}/tmp/file.gz -C {pod_path}/tmp"
+            )
             _ocp.exec_oc_cmd(rsh_cmd)
-            rsh_cmd = f"exec {pod_name} -- sync"
+
+            for x in range(copies):
+                rsh_cmd = f"exec {pod_name} -- mkdir -p {pod_path}/folder{x}"
+                _ocp.exec_oc_cmd(rsh_cmd)
+                rsh_cmd = (
+                    f"exec {pod_name} -- cp -r {pod_path}/tmp {pod_path}/folder{x}"
+                )
+                _ocp.exec_oc_cmd(rsh_cmd)
+                rsh_cmd = f"exec {pod_name} -- sync"
+                _ocp.exec_oc_cmd(rsh_cmd)
+
+            logging.info("Getting the amount of data written to the PVC")
+            rsh_cmd = f"exec {pod_name} -- df -h {pod_path}"
+            data_written = _ocp.exec_oc_cmd(rsh_cmd).split()[-4]
+            logging.info(
+                f"The Amount of data that was written to the pod is {data_written}"
+            )
+            logging.info("Deleting the pod")
+            rsh_cmd = f"delete pod {pod_name}"
             _ocp.exec_oc_cmd(rsh_cmd)
 
-        log.info("Getting the amount of data written to the PVC")
-        rsh_cmd = f"exec {pod_name} -- df -h {pod_path}"
-        data_written = _ocp.exec_oc_cmd(rsh_cmd).split()[-4]
-        log.info(f"The Amount of data that was written to the pod is {data_written}")
-        rsh_cmd = f"delete pod {pod_name}"
-        _ocp.exec_oc_cmd(rsh_cmd)
-
-        logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_two}")
-
-        pod_obj2 = helpers.create_pod(
-            interface_type=self.interface,
-            pvc_name=pvc_obj.name,
-            namespace=pvc_obj.namespace,
-            node_name=node_two,
-            pod_dict_path=constants.PERF_POD_YAML,
-        )
+            logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_two}")
 
-        start_time = time.time()
+            try:
+                pod_obj2 = helpers.create_pod(
+                    interface_type=self.interface,
+                    pvc_name=pvc_obj.name,
+                    namespace=pvc_obj.namespace,
+                    node_name=node_two,
+                    pod_dict_path=constants.PERF_POD_YAML,
+                )
+            except Exception as e:
+                logging.error(
+                    f"Pod on PVC {pvc_obj.name} was not created, exception {str(e)}"
+                )
+                raise Exception("Test was not completed")
```

Comment:
@ypersky1980 Can you add/use specific exception in this file https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/exceptions.py which will be more specific. Vasu mentioned that having general Exception might cause failure with Python3.8
Eg: Based on his comment I updated the code as in this line https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/scale/test_scale_12_OCS_worker_nodes_and_6000_PVCs.py#L97